### PR TITLE
adapterremoval nf-test snapshot

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

**Review by kpadm — COMMENTED** [review]
Looks good to me! @sateeshperi

File: modules/nf-core/adapterremoval/tests/nextflow.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+process {
```
Comment (SPPearce) [inline modules/nf-core/adapterremoval/tests/nextflow.config:1]: We don't need the empty config?

**Review by SPPearce — APPROVED** [review]
Seems to have an empty config file, but fine apart from that

File: modules/nf-core/adapterremoval/tests/nextflow.config (hunk 1 above)
Comment (sateeshperi) [reply modules/nf-core/adapterremoval/tests/nextflow.config:1]: don't need an empty file

**Comment by sateeshperi** [comment]
thank you @kpadm @SPPearce

**Pull request merged**